roles/sap_netweaver_preconfigure/tests/run-sap_netweaver_preconfigure-tests.py

```diff
@@ -40,99 +40,58 @@
     },
     {
         'number': '2',
-        'name': 'Run in assert mode on new system. Ignore a final error.',
+        'name': 'Run in assert mode on new system, \
+                 ignore assert errors.',
         'command_line_parameter': '',
         'ignore_error_final': True,
         'compact_assert_output': False,
         'rc': '99',
         'role_vars': [
             {
-                'sap_netweaver_preconfigure_assert': True
+                'sap_netweaver_preconfigure_assert': True,
+                'sap_netweaver_preconfigure_assert_ignore_errors': True,
```

Comment:
End of Python `dict` data type final key:value pair should not have comma?

Reply:
I am happy to consider all of these comments closed, if you are sure of forward compatibility. The System Python should always be CPython 3.6 for RHEL 8.x and SLES 15.x, and CPython 3.9 for RHEL 9.x? I think the trailing comma will not have any impact, there is probably a rule about this in PEP 8 though